ecl/network/v2/load_balancers/requests.go

```diff
@@ -0,0 +1,183 @@
+	DefaultGateway     string `q:"default_gateway"`
+	Description        string `q:"description"`
+	ID                 string `q:"id"`
+	LoadBalancerPlanId string `q:"load_balancer_plan_id"`
+	Name               string `q:"name"`
+	Status             string `q:"status"`
+	TenantID           string `q:"tenant_id"`
+	UserUsername       string `q:"user_username"`
+}
+
+// ToLoadBalancersListQuery formats a ListOpts into a query string.
+func (opts ListOpts) ToLoadBalancersListQuery() (string, error) {
+	q, err := eclcloud.BuildQueryString(opts)
+	return q.String(), err
+}
+
+// List returns a Pager which allows you to iterate over a collection of
+// Load Balancers. It accepts a ListOpts struct, which allows you to filter and sort
+// the returned collection for greater efficiency.
+//
+// Default policy settings return only those Load Balancers that are owned by the tenant
+// who submits the request, unless the request is submitted by a user with
+// administrative rights.
+func List(c *eclcloud.ServiceClient, opts ListOpts) pagination.Pager {
+	url := listURL(c)
+	query, err := opts.ToLoadBalancersListQuery()
+	if err != nil {
+		return pagination.Pager{Err: err}
+	}
+	url += query
+	return pagination.NewPager(c, url, func(r pagination.PageResult) pagination.Page {
+		return LoadBalancerPage{pagination.LinkedPageBase{PageResult: r}}
+	})
+}
+
+// Get retrieves a specific Load Balancer based on its unique ID.
+func Get(c *eclcloud.ServiceClient, id string) (r GetResult) {
+	_, r.Err = c.Get(getURL(c, id), &r.Body, nil)
+	return
+}
+
+// CreateOpts represents the attributes used when creating a new Load Balancer.
+type CreateOpts struct {
+
+	// AvailabilityZone is one of the Virtual Server (Nova)’s availability zone.
+	AvailabilityZone string `json:"availability_zone,omitempty"`
+
+	// Description is description
+	Description string `json:"description,omitempty"`
+
+	// LoadBalancerPlanID is the UUID of Load Balancer Plan.
+	LoadBalancerPlanID string `json:"load_balancer_plan_id" required:"true"`
+
+	// Name is a human-readable name of the Load Balancer.
+	Name string `json:"name,omitempty"`
+
+	// The UUID of the project who owns the Load Balancer. Only administrative users
+	// can specify a project UUID other than their own.
+	TenantID string `json:"tenant_id,omitempty"`
+}
+
+// ToLoadBalancerCreateMap builds a request body from CreateOpts.
+func (opts CreateOpts) ToLoadBalancerCreateMap() (map[string]interface{}, error) {
+	b, err := eclcloud.BuildRequestBody(opts, "load_balancer")
+	if err != nil {
+		return nil, err
+	}
+
+	return b, nil
+}
+
+// Create accepts a CreateOpts struct and creates a new Load Balancer using the values
+// provided. You must remember to provide a valid LoadBalancerPlanID.
+func Create(c *eclcloud.ServiceClient, opts CreateOpts) (r CreateResult) {
+	b, err := opts.ToLoadBalancerCreateMap()
+	if err != nil {
+		r.Err = err
+		return
+	}
+	_, r.Err = c.Post(createURL(c), b, &r.Body, nil)
```

Comment:
You need to set OKCodes because this endpoint returns 200 as success code. It is difficult from default codes of POST method. Please look at `defaultOkCodes` function.
https://ecl.ntt.com/en/documents/api-references/network/load_balancer/load_balancer.html#id9
```suggestion
	_, r.Err = c.Post(createURL(c), b, &r.Body, &eclcloud.RequestOpts{
		OkCodes: []int{200},
	})
```

Reply:
Thank you.
There are same problems in below so I will fix them:
  load_balancer_syslog_servers.Create
  load_balancer_actions.Reboot
  load_balancer_actions.ResetPassword